### ISSUE TITLE
Light service class class org.codinjutsu.tools.jenkins.logic.JenkinsNotifier must be final

### ISSUE THREAD

**Issue by channingwalton** [MCMicS/jenkins-control-plugin#289]
Hi,
got the error below using the plugin against Jenkins 2.

IntelliJ IDEA 2021.2 EAP (Community Edition)
Build #IC-212.4037.9, built on June 8, 2021
Runtime version: 11.0.11+9-b1481.1 x86_64
macOS 11.4
Jenkins Plugin 0.13.11-2020.3

```
com.intellij.diagnostic.PluginException: Light service class class org.codinjutsu.tools.jenkins.logic.JenkinsNotifier must be final [Plugin: Jenkins Control Plugin]
	at com.intellij.ide.plugins.PluginManagerCore.createPluginException(PluginManagerCore.java:277)
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:12)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:81)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:615)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:606)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:563)
	at com.intellij.openapi.components.ServiceManager.getService(ServiceManager.java:27)
	at org.codinjutsu.tools.jenkins.logic.JenkinsNotifier.getInstance(JenkinsNotifier.java:24)
	at org.codinjutsu.tools.jenkins.view.BrowserPanel.notifyErrorJenkinsToolWindow(BrowserPanel.java:266)
	at org.codinjutsu.tools.jenkins.logic.BrowserPanelAuthenticationHandler.loginFailed(BrowserPanelAuthenticationHandler.java:50)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeMethod(MessageBusImpl.java:645)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeListener(MessageBusImpl.java:620)
	at com.intellij.util.messages.impl.MessageBusImpl.deliverMessage(MessageBusImpl.java:417)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:390)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:372)
	at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:33)
	at com.intellij.util.messages.impl.MessageBusImpl$MessagePublisher.invoke(MessageBusImpl.java:179)
	at com.sun.proxy.$Proxy188.loginFailed(Unknown Source)
	at org.codinjutsu.tools.jenkins.logic.LoginService$1.run(LoginService.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:450)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:117)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:510)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:243)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:183)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:243)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by MCMicS** [comment]
@channingwalton Could you please try latest eap version with 2021.2 support. Available at marketplace eap channel

**Comment by channingwalton** [comment]
@MCMicS It looks like I have the latest, it was built on the 8th, and Intellij says there aren't any updates on the EAP channel.

IntelliJ IDEA 2021.2 EAP (Community Edition)
Build #IC-212.4037.9, built on June 8, 2021

**Comment by MCMicS** [comment]
I mean my plugin: https://plugins.jetbrains.com/plugin/6110-jenkins-control/versions/eap/122900

**Comment by channingwalton** [comment]
That worked. Thanks

**Comment by MCMicS** [comment]
Thanks for feedback. Will released it next days or at least when 2021.2 is released

**Comment by MCMicS** [comment]
released with [0.13.12](../releases/tag/v0.13.12)
